pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountHLLAggregationFunction.java

```diff
@@ -35,9 +36,20 @@
 
 public class DistinctCountHLLAggregationFunction extends BaseSingleInputAggregationFunction<HyperLogLog, Long> {
   public static final int DEFAULT_LOG2M = 8;
+  protected final int _log2M;
 
-  public DistinctCountHLLAggregationFunction(String column) {
-    super(column);
+  public DistinctCountHLLAggregationFunction(List<String> arguments) {
+    super(arguments.get(0));
+    int numExpressions = arguments.size();
+    // This function expects 1 or 2 arguments.
+    Preconditions
+        .checkArgument(numExpressions <= 2 && numExpressions >= 1, "DistinctCountHLL expects 1 or 2 arguments, got: ",
+            numExpressions);
+    if (arguments.size() == 2) {
+      _log2M = Integer.valueOf(arguments.get(1).replace("'", ""));
```

Comment:
No need to replace single quote here

Reply:
Integer.valueOf will throw exception on string '1'

Reply:
But why would someone put `'1'` here? In order to put `'1'` as the literal, you need to explicitly escape `'` (i.e. `DistinctCountHLL(column, '''1''')`). Also, in that case we should fail the query because it is invalid.

Reply:
from sql side, we are still doing `DistinctCountHLL(column, 1)`, this expression is parsed to long literal, then converted to string literal in BrokerRequest with single quotes.

Reply:
This doesn't seem right.. Can we add a TODO and fix it later?

Reply:
Added comments.

Currently `PinotQuery2BrokerRequestConverter` enforces single quoted non-string literal in `ParserUtils.standardizeExpression(...)`.